internal/utils/utils_test.go

```diff
@@ -108,3 +111,54 @@ func TestFilterSlice5(t *testing.T) {
 
 	assertEqual(t, res, res)
 }
+
+//
+// ReadFileInLoop
+//
+const loopFileName = "/test_loop_file.txt"
```

Comment:
```suggestion
const loopFileName = "./test_loop_file.txt"
```
В корне диска не очень хорошо файлы создавать)

Reply:
Это уже моя оЧепятка, собирался рядом с исполняемым файлом естественно, спасибо